techniques/css/C37.html

```diff
@@ -2,86 +2,66 @@
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to be able to present images without introducing
-               a horizontal scroll bar at a width equivalent to 320 CSS pixels, or a vertical scroll
-               bar at a height equivalent to 256 CSS pixels for content intended to scroll horizontally.
-               This is done by using CSS <code>max-width</code> and <code>height</code> property techniques that adapt to the available space and keep the original dimensions
-               of the image.
-            </p>
-            <p>Responsive layouts can add or remove columns or layout blocks, and each part of the
-               layout can be wider or smaller at different points. This technique ensures images
-               do not break out of their layout area, including in one-column layouts where it would
-               cause scrolling.
-            </p>
-            <p>The basic principles of fitting images are to:</p>
+            <h2>解説</h2>
+            <p>この達成方法の目的は、320 CSS ピクセルに相当する幅において横スクロールバーを、又は 256 CSS ピクセルに相当する高さにおいて縦スクロールバーを導入することなく、画像を表示できるようにすることである。これは、画像の元のサイズを維持しつつ使用可能なスペースに適応させる、CSS の <code>max-width</code> 及び <code>height</code> プロパティ手法を用いることによって行われる。</p>
+            <p>レスポンシブなレイアウトでは、カラムやレイアウトブロックを追加したり削除したりできる。また、レイアウトの各部分は、さまざまなポイントで幅を広くしたり小さくしたりできる。この手法により、スクロールが発生する1カラムのレイアウトを含めて、画像がレイアウト領域からはみ出すことがなくなる。</p>
+            <p>画像を適応させるための基本原則は次のとおりである:</p>
             <ol>
                
-               <li>Define the <code>max-width</code> property for images, and;
+               <li>画像の <code>max-width</code> プロパティを定義する。
                </li>
                
-               <li>Define the <code>height</code> property for images, so they enlarge or shrink in the available space and respond
-                  to zoom levels.
+               <li> 画像の <code>height</code> プロパティを定義して、使用可能なスペースで画像を拡大又は縮小し、ズームレベルに対応させる。  
                </li>
                
             </ol>
-            <p>All images require design finesse by making sure the original size fits the biggest
-               size of the available spaces to achieve good-looking results at a wide range of viewport
-               sizes and zoom levels.
+            <p>全ての画像は、さまざまなビューポートサイズとズームレベルで綺麗な見ためになるようにするために、元のサイズが利用可能なスペースの最大サイズに確実に適応させるようにすることによる、デザインの精巧さを必要とする。
             </p>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-fitting-images-in-html-and-css">
-               <h3>Example 1: Fitting images in HTML and CSS</h3>
-               <p>The following simple example uses HTML and CSS to create a fitting image. The layout
-                  regions adjust their size as the viewport is adjusted. The images subsequently adjust
-                  their size to fit within the layout region containers.
+               <h3>例 1: HTML と CSS での画像の適応</h3>
```

Comment:
[fitting](https://eow.alc.co.jp/search?q=fitting) を形容詞と取ってみるとか

```suggestion
               <h3>例 1: HTML 及び CSS での適切な画像</h3>
```